includes/class-amp-story-post-type.php

```diff
@@ -900,19 +930,19 @@ public static function the_single_story_card( $post ) {
 		);
 
 		?>
-		<a class="latest_stories__link" href="<?php echo esc_url( get_permalink( $post ) ); ?>">
+		<<?php echo esc_attr( $wrapper_tag_name ); ?> href="<?php echo $args['disable_link'] ? '' : esc_url( get_permalink( $args['post'] ) ); ?>" class="latest_stories__link">
```

Comment:
In the Latest Stories block in the editor, this prevents being able to click the story. It looks like the Latest Posts block from Core has links that are clickable.